src/otx/algo/action_classification/heads/movinet_head.py

```diff
@@ -33,12 +32,20 @@ def __init__(
         num_classes: int,
         in_channels: int,
         hidden_dim: int,
-        loss_cls: dict,
+        loss_cls: nn.Module,
```

Comment:
Could you update and add docstring for `loss_cls` and `topk` argument?

Reply:
fixed at 36e79c2fc6f88460f922d3eec9b6fe1e01475db0